source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -771,6 +757,21 @@ void Filter::addDynamicMetadata(const ProcessorState& state, ProcessingRequest&
   *req.mutable_metadata_context() = forwarding_metadata;
 }
 
+void Filter::addAttributes(ProcessorState& state, ProcessingRequest& req) {
+  if (!state.sendAttributes(config_->expressionManager())) {
+    return;
+  }
+
+  auto activation_ptr = Filters::Common::Expr::createActivation(
+      &config_->expressionManager().localInfo(), state.callbacks()->streamInfo(),
+      state.requestHeaders(), dynamic_cast<const Http::ResponseHeaderMap*>(state.responseHeaders()),
```

Comment:
@jbohanon 
when you pass the response header to createActivation, is there guarantee that it won't be mutated? 
const RequestOrResponseHeaderMap* ptr can still be pointed to a different location?